docs/Changelog.md

```diff
@@ -424,7 +424,7 @@ opset_import {
 
 <dl>
 <dt><tt>X</tt> : T</dt>
-<dd>The input 4-dimensional tensor of shape NCHW.</dd>
+<dd>The input 4-dimensional tensor of shape NCHW for 2D BatchNormalization or 5-dimensional tensor of shape NCHWD for 3D BatchNormalization.</dd>
```

Comment:
Thank you for the change/fix! I think 3-dimensional input tensor of shape NCH (probably NCI) should also be allowed :).

Reply:
Yep, we can say that tensor is r-dimensional as normalization on all axises >=2 is applied